llama_index/indices/managed/vectara/retriever.py

```diff
@@ -67,7 +69,7 @@ def __init__(
         summary_enabled: bool = False,
         summary_response_lang: str = "eng",
         summary_num_results: int = 7,
-        summary_prompt_name: str = "vectara-summary-ext-v1.2.0",
+        summary_prompt_name: str = "vectara-experimental-summary-ext-2023-10-23-small",
```

Comment:
is this a parameters for Vectara? should we put a available list for this variable instead of hardcode it. you can use this one as default

Reply:
Yes we have essentially 4 options today: https://docs.vectara.com/docs/learn/grounded-generation/select-a-summarizer - what's your recommendation? do it just as a comment, maybe link to this page, or something else?

Reply:
can make a comment and link to the page